Doc/library/lzma.rst

```diff
@@ -311,6 +311,25 @@ Compressing and decompressing data in memory
    *preset* and *filters* arguments.
 
 
+.. function:: crc32(data[, value])
+
+   .. index::
+      single: Cyclic Redundancy Check
+      single: checksum; Cyclic Redundancy Check
+
+   Computes a CRC (Cyclic Redundancy Check) checksum of *data*. The
+   result is an unsigned 32-bit integer. If *value* is present, it is used
```

Comment:
Python only has `int`, so perhaps: 
```suggestion
   result is a positive integer, in the range ``0 <= result < 2**32``.
   If *value* is present, it is used
```

This is more verbose, though, and the [`zlib.crc32`](https://docs.python.org/dev/library/zlib.html#zlib.crc32) wording just says "unsigned 32-bit integer".